src/main/cpp/subsystems/Wrist.cpp

```diff
@@ -2,29 +2,31 @@
 
 #include "frc2/command/RunCommand.h"
 
-Wrist::Wrist() {
-    MotorUtils::Motor::LogValues logValues {true, true, true};
+Wrist::Wrist()
+{
+    MotorUtils::Motor::LogValues logValues{true, true, true};
     MotorUtils::Motor wristMotor{&m_motor, CONSTANTS::WRIST::PidValue, logValues};
     AddPID(wristMotor);
     SetPID();
 
     m_motor.SetPosition(CONSTANTS::WRIST::DEFAULT_POSITION);
 }
 
-void Wrist::set_angle(units::angle::degree_t angle) {
-    m_motor.SetControl(ctre::phoenix6::controls::PositionVoltage {angle});
+void Wrist::set_angle(units::angle::degree_t angle)
+{
+    m_motor.SetControl(ctre::phoenix6::controls::PositionVoltage{angle});
```

Comment:
The control object (ctre::phoenix6::controls::PositionVoltage) should be created once in Wrist.h. Then here, call m_motor.SetControl(your_control_obj.WithPosition(angle))